src/Components/ServiceScene/Breakdowns/LabelBreakdownScene.tsx

```diff
@@ -341,11 +349,44 @@ function buildLabelsLayout(options: VariableValueOption[], scene: LabelBreakdown
 }
 
 function getExpr(tagKey: string) {
-  return `sum(count_over_time(${LOG_STREAM_SELECTOR_EXPR} | drop __error__ | ${tagKey}!="" [$__auto])) by (${tagKey})`;
+  // Is there a better way to check for all detected metadata?
+  if (tagKey === LEVEL_NAME) {
+    return `sum(count_over_time(${VAR_LABELS_EXPR} | ${LEVEL_NAME}!="" [$__auto])) by (${tagKey})`;
+  }
+  return `sum(count_over_time(${VAR_LABELS_EXPR} [$__auto])) by (${tagKey})`;
```

Comment:
We would lose filtering by lines, fields and patterns with that. Not sure that is foreseen?